src/utils/dispute-utils.js

```diff
@@ -208,26 +225,36 @@ export function getPhaseAndTransition(dispute, courtConfig, nowDate) {
   }
 }
 
+/**
+ * Tells the adjudication state of a round at a certain time.
+ * @param {Object} dispute Dispute querying the adjudication round of
+ * @param {Object} round The round that is being queried
+ * @param {Number} now Current time in ms
+ * @param {Object} courtConfig The court configuration
+ * @returns {Object} The adjudication phase of the requested round at the given time
+ */
 export function getAdjudicationPhase(dispute, round, now, courtConfig) {
   const {
     termDuration,
     commitTerms,
     revealTerms,
     appealTerms,
     appealConfirmationTerms,
-    maxRegularAppealRounds,
   } = courtConfig
-  const numberOfRounds = dispute.rounds.length
+
   const { draftTermId, delayedTerms, number: roundId } = round
 
   const drafTermStartTime = getTermStartTime(draftTermId, courtConfig)
```

Comment:
```suggestion
  const draftTermStartTime = getTermStartTime(draftTermId, courtConfig)
```

Small misspelling :)